exercise/templatetags/exercise.py

```diff
@@ -223,8 +223,11 @@ def min_group_size(context):
 
 @register.simple_tag(takes_context=True)
 def module_accessible(context, entry):
-    t = entry.get('reading_opening_time', entry.get('opening_time'))
-    return _is_accessible(context, entry, t)
+    t = entry.get('reading_opening_time')
```

Comment:
Does this really change anything? Looks like it does the same stuff as the old code.

Reply:
The old code didn't work, if `reading_opening_time` was not set. I think it was maybe the nested get method that caused malfunction. (The outer get method would always find the object even if it was empty.)

Reply:
But the logic should be same, and the changed version works as it should.